spec/instant.html

```diff
@@ -567,7 +559,7 @@ <h1>AddInstant ( _epochNanoseconds_, _hours_, _minutes_, _seconds_, _millisecond
             _seconds_ × 1,000,000,000 +
             _minutes_ × 60,000,000,000 +
             _hours_ × 3,600,000,000,000.
```

Comment:
`AddInstant` needs to add type assertions to its arguments, and also be explicit about which kind of numbers are being used in this equation.